api/storage/elasticsearch/elasticsearch.go

```diff
@@ -304,7 +304,18 @@ func (sm *StorageManager) GetResources(resourceType string, executionID string,
 	searchResult, err := sm.client.Search().
```

Comment:
Don't you think it will be more readable to change the searchResult to searchResultTotalHits?
Because you actually don't care about the search results in this case (size 0 )

Reply:
No, **but** if it is not readable to you, it can be not readable to the others. 
so I have changed it :)